### PR TITLE
Fix for different versions of python/pandas/eurostat

### PR DESCRIPTION
**Pull request by jankrcal**
This PR makes two minor fixes that makes the notebooks run for me :)

### CODE REVIEW

**Comment by jankrcal** [comment]
@matejpiro Could you please check if this still works for you or breaks your code?

**Comment by jankrcal** [comment]
@mgrabovsky Could you PTAL?

File: notebooks/regional_emissions_by_category_lib/emission_plots.py [hunk 1]
```diff
@@ -13,11 +13,16 @@ def _get_data(state, year):
     """
     df = eurostat.get_data_df('env_air_gge')
     # Pandas query() does not allow backslash in column names so "rename column" is needed.
+    # In some versions of data, the column is called geo\time, in some geo\TIME_PERIOD.
     df = df.rename(columns={'geo\\time': 'geo'})
+    df = df.rename(columns={'geo\\TIME_PERIOD': 'geo'})
```
Comment (mgrabovsky) [inline notebooks/regional_emissions_by_category_lib/emission_plots.py:18]: These two lines can be merged (with the dict having two entries.

**Review by mgrabovsky — APPROVED** [review]
Looks good. One minor suggestion for improvement.

File: notebooks/regional_emissions_by_category_lib/emission_plots.py (hunk 1 above)
Comment (jankrcal) [reply notebooks/regional_emissions_by_category_lib/emission_plots.py:18]: Thanks, done.

**Pull request merged**